projects/plugins/jetpack/modules/search/instant-search/store/effects.js

```diff
@@ -14,6 +14,8 @@ import {
 	setSort,
 } from './actions';
 
+let RESPONSE_ID = 0;
```

Comment:
Should we be all-caps-ing this when it's not a constant?

Reply:
Changed to `responseCounter`.